hpo_demo_setup.sh

```diff
@@ -0,0 +1,288 @@
+		fi
+		check_err "ERROR: git clone of kruize/benchmarks failed."
+	fi
+	echo "done"
+	echo "#######################################"
+	echo
+}
+
+###########################################
+#   Cleanup HPO git Repos
+###########################################
+function delete_repos() {
+	echo "1. Delete hpo and benchmarks git repos"
+	rm -rf hpo benchmarks
+}
+
+###########################################
+#   Start HPO
+###########################################
+function hpo_install() {
+	echo
+	echo "#######################################"
+	echo "6. Start HPO"
+	if [ ! -d hpo ]; then
+		echo "ERROR: hpo dir not found."
+		if [ ${hpo_restart} -eq 1 ]; then
+			echo "ERROR: HPO not running. Wrong use of restart command"
+		fi
+		exit -1
+	fi
+	pushd hpo >/dev/null
+		if [ -z "${HPO_DOCKER_IMAGE}" ]; then
+			HPO_DOCKER_IMAGE=${HPO_DOCKER_REPO}:${HPO_VERSION}
+		fi
+		if [[ ${CLUSTER_TYPE} == "native" ]]; then
+			echo
+			echo "Starting hpo with  ./deploy_hpo.sh -c ${CLUSTER_TYPE}"
+			echo
+			./deploy_hpo.sh -c ${CLUSTER_TYPE} >> ${LOGFILE}  &
+			#check_err "ERROR: HPO failed to start, exiting"
+		else
+			echo
+			echo "Starting hpo with  ./deploy_hpo.sh -c ${CLUSTER_TYPE} -h ${HPO_DOCKER_IMAGE}"
+			echo
+
+			./deploy_hpo.sh -c "${CLUSTER_TYPE}" -h "${HPO_DOCKER_IMAGE}"
+			#check_err "ERROR: HPO failed to start, exiting"
+		fi
+	popd >/dev/null
+	echo "#######################################"
+	echo
+}
+
+###########################################
+#   Start HPO Experiments
+###########################################
+function hpo_experiments() {
+
+	SEARCHSPACE_JSON="hpo_helpers/search_space.json"
+	URL="http://localhost:8085"
+	## TODO : Add check to eid
+	eid=$(${PY_CMD} -c "import hpo_helpers.utils; hpo_helpers.utils.getexperimentid(\"${SEARCHSPACE_JSON}\")")
+	exp_output="experiment-output.csv"
+
+	##TODO : Delete the old logs if any before starting the experiment
+
+	echo "#######################################"
+	echo "Start a new experiment with search space json"
+	## Step 1 : Start a new experiment with provided search space.
+	exp_json=$( cat ${SEARCHSPACE_JSON} )
+	if [[ ${exp_json} != "" ]]; then
+		curl -Ss -H 'Content-Type: application/json' ${URL}/experiment_trials -d '{ "operation": "EXP_TRIAL_GENERATE_NEW",  "search_space": '"${exp_json}"'}'
+		check_err "Error: Creating the new experiment failed."
+	else
+		check_err "Error: Searchspace is empty"
+	fi
+
+	## Looping through trials of an experiment
+	for (( i=0 ; i<${N_TRIALS} ; i++ ))
+	do
+		## Step 2: Get the HPO config from HPOaaS
+		echo "#######################################"
+        	echo "Generate the config for trial ${i}"
+		sleep 2
+		HPO_CONFIG=$(curl -Ss -H 'Accept: application/json' "${URL}"'/experiment_trials?experiment_id='"${eid}"'&trial_number='"${i}")
+		#echo ${HPO_CONFIG}
+		if [[ ${HPO_CONFIG} != -1 ]]; then
+			echo "${HPO_CONFIG}" > hpo_config.json
+		else
+			check_err "Error: Issue generating the configuration from HPO."
+		fi
```

Comment:
Same here, dont need the else
if [[ ${HPO_CONFIG} == -1 ]]; then
      check_err "Error: Issue generating the configuration from HPO."
fi
echo "${HPO_CONFIG}" > hpo_config.json